### ISSUE TITLE
Autopause messes up on websites where music loads too fast.

### ISSUE THREAD

**Issue by WarmMillk** [VolcanicArts/VRCOSC#68]
On Youtube its not an issue since the autopause that the VRCOSC has is short enough and Youtube takes forever to load the next video. On something like SoundCloud, at least for me, skipping a song/leaving it to switch to the next will start the next song, then autopauses. This issue only started during the new update

**Comment by VolcanicArts** [comment]
I've got a few ideas on why that could happen. I'll investigated tomorrow

**Comment by VolcanicArts** [comment]
The reason I do that is to force a media update. I can probably find a different way. No idea how that would cause autopausing, but I hadn't tested SoundCloud

**Comment by WarmMillk** [comment]
The way you had it previously forcing media updates worked perfectly fine, it never paused music and would work seemlessly

**Comment by VolcanicArts** [comment]
I've found the issue.

The way I was forcing a media update before was to always try and play the media source on connection, however that had problems if the media was already playing as there would be no update. I changed to instead check to see if the media was paused or playing, and do the opposite before reverting so it seemed like nothing had changed in the perspective of the user. Problem is, if a website loads too quickly then the media is registered as paused, with which I then play and then pause. However YouTube and other sites will immediately play after loading, which VRCOSC doesn't take into account and therefore pauses after it tries to play.

I can either revert back to the previous solution, or delay the check, of which I think I'll go with the former to avoid confusion
